locales/en/population.json

```diff
@@ -40,5 +40,10 @@
   "breadcrumbs": {
     "home": "Population Overview"
   },
-  "no_population": "No population to display"
+  "no_population": "No population to display",
+  "no_transfers": "None booked",
```

Comment:
Would this be better as `No transfer booked`? `None` could be ambiguous, unless we have something clear in the markup surrounding where this is used?

Reply:
I did have it as "No transfer booked" for a while, but it wrapped, and also it didn't agree with the mockups.

Given Andy is unavailable for the next few weeks, I'd prefer to stick as close to the design as possible for now, and then fix these sorts of issues when he's back.